DFE-Digital/early-years-foundation-reform
# HFEYP-610 Enable User feedback

## Ticket and context

Ticket: [HFEYP-610](https://dfedigital.atlassian.net/browse/HFEYP-610)

## Acceptance criteria
### Feedback panel

- Users can view the Feedback panel across all website page footer as per design from ife.akinbolaji 
- ‘Give feedback’ button should take user to the Google form[Help for Early Years Providers ]
- User click on ‘Yes’ button should display the message “Thank you for your feedback”
- User click on ‘No’ should expand the form with options to:
  - view the text as per wireframe
  - “Close” button to close the form and return to original state, and retain focus on the ‘No’ button as per screenshot
  - and “Give feedback” to take user to the Google form [Help for Early Years Providers ] in a new tab, and retain focus on the “Give feedback” button

## Tech review
There is a little tiding up of the spacing in layouts included.

### Code quality checks

### How can someone see it in review app?
1. Click the link to review app (posted by a `github-actions` bot below)
2. The feedback form should be visible at the bottom of the page. Test the functionality described above.
3. Navigate to another page and make sure the functionality works there too.
